src/messaging/ExchangeContext.cpp

```diff
@@ -266,6 +267,27 @@ CHIP_ERROR ExchangeContext::SendMessage(Protocols::Id protocolId, uint8_t msgTyp
     }
 }
 
+void ExchangeContext::SendStatusReport(const Protocols::SecureChannel::StatusReport & statusReport)
+{
+    ChipLogDetail(ExchangeManager, "Sending status report. Protocol code %u, exchange %u", statusReport.GetProtocolCode(),
```

Comment:
This should probably be a progress log, if not error; these are fairly rare and generally indicate error conditions....

Reply:
Ah, good point.  I'd still say progress.  Status reports are pretty rare.